src/app/pages/elements/chips/ChipsDemo.ts

```diff
@@ -54,142 +55,239 @@ const template = `
-        };
-        this.async = {
-            options: () => {
-                return new Promise((resolve) => {
-                    setTimeout(() => {
-                        resolve(abbrieviated);
-                    }, 300);
-                });
+  constructor() {
+    let states = [
+      'Alabama',
+      'Alaska',
+      'Arizona',
+      'Arkansas',
+      'California',
+      'Colorado',
+      'Connecticut',
+      'Delaware',
+      'Florida',
+      'Georgia',
+      'Hawaii',
+      'Idaho',
+      'Illinois',
+      'Indiana',
+      'Iowa',
+      'Kansas',
+      'Kentucky',
+      'Louisiana',
+      'Maine',
+      'Maryland',
+      'Massachusetts',
+      'Michigan',
+      'Minnesota',
+      'Mississippi',
+      'Missouri',
+      'Montana',
+      'Nebraska',
+      'Nevada',
+      'New Hampshire',
+      'New Jersey',
+      'New Mexico',
+      'New York',
+      'North Dakota',
+      'North Carolina',
+      'Ohio',
+      'Oklahoma',
+      'Oregon',
+      'Pennsylvania',
+      'Rhode Island',
+      'South Carolina',
+      'South Dakota',
+      'Tennessee',
+      'Texas',
+      'Utah',
+      'Vermont',
+      'Virginia',
+      'Washington',
+      'West Virginia',
+      'Wisconsin',
+      'Wyoming',
+    ];
+    let abbrieviated = [
+      {
+        value: 'USA',
+        label: 'United States',
+      },
+      {
+        value: 'GB',
+        label: 'Great Britain',
+      },
+      {
+        value: 'CA',
+        label: 'Canada',
+      },
+      {
+        value: 'AU',
+        label: 'Australia',
+      },
+    ];
+    let collaborators = [
+      {
+        id: 1,
+        firstName: 'Brian',
+        lastName: 'Kimball',
+      },
+      {
+        id: 2,
+        firstName: 'Josh',
+        lastName: 'Godi',
+      },
+      {
+        id: 3,
+        firstName: 'Alec',
+        lastName: 'Sibilia',
+      },
+      {
+        id: 4,
+        firstName: 'Kameron',
+        lastName: 'Sween',
+      },
+    ];
+    this.staticDemo = { options: states };
+    this.formatted = {
+      format: '$firstName $lastName',
+      options: collaborators,
+    };
+    this.async = {
+      options: () => {
+        return new Promise((resolve) => {
+          setTimeout(() => {
+            resolve(abbrieviated);
+          }, 300);
+        });
+      },
+      getLabels: (data) => {
+        return new Promise((resolve) => {
+          setTimeout(() => {
+            let values = data.map((item) => item.value);
+            let results = abbrieviated.filter((item) => values.includes(item.value));
+            resolve(results);
+          }, 300);
+        });
+      },
+    };
+    this.avalue = [
+      {
+        value: 'USA',
+      },
+      {
+        value: 'GB',
+      },
+    ];
+    this.rowValue = [
+      {
+        id: 1,
+        firstName: 'Brian',
+        lastName: 'Kimball',
+      },
+    ];
+    this.rowDemo = {
+      ...this.formatted,
+      columns: {
```

Comment:
I don't understand the structure of this config.  Maybe we should think about it if could be cleaner.  Also wondering if we should just transclude a template into it.